src/sdk/pynni/nni/compression/torch/compressor.py

```diff
@@ -308,11 +316,13 @@ def _instrument_layer(self, layer, config):
         assert "quant_types" in config, 'must provide quant_types in config'
         assert isinstance(config["quant_types"], list), 'quant_types must be list type'
         assert "quant_bits" in config, 'must provide quant_bits in config'
-        assert isinstance(config["quant_bits"], int) or isinstance(config["quant_bits"], dict), 'quant_bits must be dict type or int type'
+        assert isinstance(config["quant_bits"], int) or isinstance(config["quant_bits"],
+                                                                   dict), 'quant_bits must be dict type or int type'
```

Comment:
this line break seems strange, better to break line after `dict), `

Reply:
resolved